src/app/components/cluster-info/cluster-info.component.ts

```diff
@@ -59,8 +61,13 @@ export class ClusterInfoComponent implements OnInit {
         this.updateContainerStatusData(data);
       });
 
-    this.jobHistoryData = this.generateHistoryData();
-    this.containerHistoryData = this.generateHistoryData();
+    this.scheduler.fetchAppHistory().subscribe(data => {
+      this.jobHistoryData = this.getAreaChartData(data);
```

Comment:
Can we rename this to app history data? 
Or should we do it similarly to what we did in core/shim and change from job to app in a separate jira?

Reply:
I think we can open a separate JIRA to track that, as that is not blocking the release.